### ISSUE TITLE
JSON Schema breaks valid ENV Variables

### ISSUE THREAD

**Issue by jseiser** [runatlantis/helm-charts#339]
https://github.com/runatlantis/helm-charts/blob/c86eab114b0ec1f8d0bce812d5bbc9f7baacc30b/charts/atlantis/values.schema.json#L859C1-L865C7

# Error

```
│ - environment.ATLANTIS_WRITE_GIT_CREDS: Invalid type. Expected: string, given: boolean
│ - environment.ATLANTIS_ALLOW_DRAFT_PRS: Invalid type. Expected: string, given: boolean
```

# locals.tf
```
  environment_variables = {
    ATLANTIS_DEFAULT_TF_VERSION = "v1.5.2"
    ATLANTIS_WRITE_GIT_CREDS    = "true"
    ATLANTIS_ALLOW_DRAFT_PRS    = "true"
  }
```

# Part of a helm_resource
```
  dynamic "set" {
    for_each = local.environment_variables

    content {
      name  = format("environment.%s", upper(set.key))
      value = format("%s", set.value)
    }
  }
```

# Whats rendered
```
  - environment              = {
      - ATLANTIS_ALLOW_DRAFT_PRS    = true
      - ATLANTIS_DEFAULT_TF_VERSION = "v1.5.2"
      - ATLANTIS_WRITE_GIT_CREDS    = true
    }
```

This was working, and the docs read like its a bool: https://github.com/runatlantis/atlantis/blob/main/runatlantis.io/docs/server-configuration.md?plain=1#L1082

**Comment by jamengual** [comment]
Should we close this?

**Comment by kvanzuijlen** [comment]
@jamengual this was fixed in #338
